### DISCUSSION TITLE
How to add custom metrics to the final report.html without modifying the locust code.

### DISCUSSION THREAD

**Discussion by Alex-TG001** [locustio/locust#2776] [Q&A]
I'm trying to add my own metrics to the final front-end report page, but I can't find a way to pass parameters to locust from the outside, do you have any ideas?

**Comment by andrewbaldwin44** [comment]
Hey @Alex-TG001, right now it's not officially supported within locust to modify the HTML report page, however you could generate your own CSV file and have it downloadable from the reports tab, like in this example: https://github.com/locustio/locust/blob/master/examples/extend_web_ui.py
